index.bs

```diff
@@ -1975,6 +1979,124 @@ The following extended attributes are applicable to operations:
         "void"
 </pre>
 
+<h5 id="idl-tojson-operation">toJSON</h5>
+
+By declaring a “toJSON” [=regular operation=],
+an [=interface=] specifies how to convert the objects that implement it to [=JSON types=].
+
+The “toJSON” [=regular operation=] is reserved for this usage.
+It must take zero arguments and return a [=JSON type=].
+
+The <dfn id="dfn-json-types" export lt="JSON type">JSON types</dfn> are:
+
+*   [=numeric types=],
+*   {{boolean}},
+*   [=string types=],
+*   [=nullable types=] whose [=nullable types/inner type=] is a [=JSON type=],
+*   [=annotated types=] whose [=annotated types/inner type=] is a [=JSON type=],
+*   [=union types=] where all of their [=member types=] are [=JSON types=],
```

Comment:
Nit: you switched from "whose" to "where". I think all "whose" might work well,  e.g. "union types whose member types are all JSON types"